casepro/cases/models.py

```diff
@@ -103,6 +103,16 @@ def __str__(self):
         return self.name
 
 
+class SystemUser(User):
+
+    @classmethod
+    def get_or_create(cls):
+        systemUsers = cls.objects.filter()
+        if len(systemUsers) > 0:
+            return systemUsers[0]
```

Comment:
It doesn't create a table, it'll create a user record. There's also Django's own `get_or_create()` function that exists to deal with this stuff.